### ISSUE TITLE
When new release will be published?

### ISSUE THREAD

**Issue by cawa-93** [libass/JavascriptSubtitlesOctopus#94]
I will import JavascriptSubtitlesOctopus as a native ES module. And this causes an error when initializing:
```
Uncaught TypeError: 'caller', 'callee', and 'arguments' properties may not be accessed on strict mode functions or the arguments objects for calls to them
    at HTMLVideoElement.<anonymous> (subtitles-octopus.js:205)
```

After researching, I found out that this problem was fixed by https://github.com/libass/JavascriptSubtitlesOctopus/commit/70d499eb6b8c5c0c7143fca1dfaae8646060f97b 7 months ago. However, it is still relevant for the latest published version

**Comment by TFSThiagoBR98** [comment]
Probably this weekend

**Comment by TheOneric** [comment]
Currently SubtitlesOctopus declares the wrong licence in its `package.json` *(relevant for npm releases I assume)* and falls short of properly crediting the authors of bundled libraries, which is also violates the licences of aid bundled libraries *(relevant for all binary releases/distributions)* — and even JSO's own licence.

I believe we should hold off on a new release until these issues are fixed.

**Comment by starkana** [comment]
Thank god this lib is continued being maintained. It has so much potential. Subs are there for life let’s make them right.

**Comment by TFSThiagoBR98** [comment]
@TheOneric is probably better to open a new issue for the license problem

**Comment by SoniEx2** [comment]
What's currently blocking this? We see #99 is closed?

**Comment by starkana** [comment]
Has the licensing issue been dealt with? Would be nice for a release after 2yrs+

**Comment by AnonymusRaccoon** [comment]
I'm sorry to be bump this again, but I think updating the npm package would be a good thing. I don't want to use a submodule on a frontend project to include this library x)

**Comment by SoniEx2** [comment]
we don't wanna explain to mozilla why we're not using a proper release in a browser extension

**Comment by AnonymusRaccoon** [comment]
I'm sorry, I don't understand what you mean.

**Comment by SoniEx2** [comment]
https://extensionworkshop.com/documentation/publish/add-on-policies/#development-practices

can't use dev versions

**Comment by AnonymusRaccoon** [comment]
SubtitileOctopus is not a Firefox add-on, this is a npm package. This policy does not concern this project

**Comment by SoniEx2** [comment]
no it really does if you're trying to use it as part of an addon for any reason. see https://soniex2.autistic.space/posts/2022/03/06-occf.xhtml

**Comment by Helvio88** [comment]
I have a PR pending on this issue. Any idea when it could be approved?
